code/game/gamemodes/wizard/raginmages.dm

```diff
@@ -148,20 +151,38 @@
 			return
 		else
 			candidates = shuffle(candidates)
-			for(var/client/i in candidates)
+			for(var/mob/dead/observer/i in candidates)
 				if(!i) continue //Dont bother removing them from the list since we only grab one wizard
 
-				theclient = i
+				// YER A WIZZERD HARRY
+				harry = i
 				break
 
 			making_mage = 0
-			if(theclient)
-				var/mob/living/carbon/human/new_character= create_human_for_client_from_prefs(theclient)
+			if(harry)
+				var/mob/living/carbon/human/new_character= makeBody(harry)
 
 
 				new_character.mind.make_Wizard() // This puts them at the wizard spawn, worry not
 				mages_made++
 				return 1
+			else
+				log_to_dd("The candidates list for ragin' mages contained non-observer entries!")
+				return 0
+
+// ripped from -tg-'s wizcode, because whee lets make a very general proc for a very specific gamemode
+// This probably wouldn't do half bad as a proc in __HELPERS
+// Lemme know if this causes species to mess up spectacularly or anything
+/datum/game_mode/wizard/raginmages/proc/makeBody(var/mob/dead/observer/G)
+	if(!G || !G.key) return // Let's not steal someone's soul here
+
+	var/mob/living/carbon/human/new_character = new(pick(latejoin))
+
+	G.client.prefs.copy_to(new_character)
+
+	new_character.key = G.key
+
+	return new_character
 
```

Comment:
Not sure if this will result in anything too catastrophic - does this do everything OK?